docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx

```diff
@@ -14,30 +14,25 @@ function MyComponent() {
   const [createLivenessApiData, setCreateLivenessApiData] = React.useState(null);
   
   /* 
-   * 1. Check whether the difference between server time and device time is
-   *  greater than or equal to 5 minutes, and if so, return the offset in milliseconds.
-   *  This logic should be adjusted based on the server response and use case
```

Comment:
Are these changes related to your PR? Same comment on all the other liveness files.